visualization/app/codeCharta/state/store/treeMap/treeMap.actions.ts

```diff
@@ -23,5 +23,6 @@ export function setTreeMapSettings(treeMapSettings: RecursivePartial<TreeMapSett
 
 export const defaultTreeMapSettings: TreeMapSettings = {
 	// Plop: Append default property here
+	// TODO check fix map size. map size should be set dynamically (consider screen resolution).
```

Comment:
Is this done?

Reply:
We should think about this afterwards. Thanks for the hint.